### ISSUE TITLE
stringtie: prepDE for multiple samples

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/tools-iuc#1680]
I'm wondering if the current integration of prepDE.py in the tool is really useful. I guess most users will input a dataset list of sam/bam files where each corresponds to a sample. Currently prepDE.py is called separately for each sample. Hence it seems that it can not be user for differential expression analysis. 

A solution might be to generate a separate tool for prepDE that uses all data sets and creates subdirectories for each sample as described in https://ccb.jhu.edu/software/stringtie/index.shtml?t=manual#deseq. 

ping https://github.com/galaxyproject/tools-iuc/issues/1322

**Comment by mblue9** [comment]
Yes that was what @NCEichner was also looking for in https://github.com/galaxyproject/tools-iuc/pull/1545 

It sounds like it would be a good idea to implement, I just don't have any time to work on it right now.

**Comment by nsoranzo** [comment]
I think that having them separate is useful in order to leave the design specification inside DESeq2.
